.github/workflows/ci.yml

```
@@ -18,5 +18,5 @@ jobs:
         with:
           go-version: "1.27.1"
 
-      - name: Run tests with coverage
-        run: go test -cover ./...
+      - name: RCheck formatting
+        run: test -z $(go fmt ./...)
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:1:9: Quote this to prevent word splitting

(shellcheck)